### DISCUSSION TITLE
Mobile app blog section

### DISCUSSION THREAD

**Discussion by chrisK824** [community/community#48744] [Mobile] [answered]
Wouldn't a blog section be helpfull to get users up to date with releases, new features etc? Of course push notifications for preferences and topics.

**Comment by eliperkins** [comment, marked as answer]
Hey @chrisK824 thanks for this feedback. You can see new releases from repositories, users and orgs you follow on GitHub within the Explore tab in the app. You can also filter Explore to only include "Announcements" and "Releases" to see the content you care most about:

![Simulator Screen Shot - iPhone 14 Pro - 2023-03-06 at 08 23 54](https://user-images.githubusercontent.com/1051453/223153654-37c6aa82-7873-45e4-83f7-a2871f458053.png)

For significant updates to GitHub Mobile itself, you can check out [the `mobile` label on the GitHub Changelog](https://github.blog/changelog/label/mobile/).

> Of course push notifications for preferences and topics.
 
I'll make a note with the team about how push notifications could be helpful here. To help me better understand what you're looking for, can I ask a couple of questions?

- What repositories would you expect to get push notifications for? All repositories you watch? Updates to repositories from users or orgs you follow?
- What level of granularity would you look for on push notifications? All issues, pull requests, discussions, releases? Just a subset of those?

**Comment by chrisK824** [comment]
Push notifications would really variate, I guess based on community feedback and subscriptions but a few in mind are watched repos activity, github features and major release notes etc
